activerecord/lib/active_record/relation.rb

```diff
@@ -346,10 +346,16 @@ def scoping
     end
 
     # Updates all records in the current relation with details given. This method constructs a single SQL UPDATE
-    # statement and sends it straight to the database. It does not instantiate the involved models and it does not
-    # trigger Active Record callbacks or validations. Values passed to #update_all will not go through
-    # Active Record's type-casting behavior. It should receive only values that can be passed as-is to the SQL
-    # database.
+    # statement and sends it straight to the database.
+    #
+    # Regular update procedures are bypassed. In particular, please note that:
+    #
+    # * \Validations are skipped.
+    # * \Callbacks are skipped.
+    # * +updated_at+/+updated_on+ are not updated.
+    # * Involved models are not instantiated
+    #
+    # However, values passed to #update_all will still go through Active Record's type-casting behavior.
```

Comment:
This line makes me think it's the same as `.attributes = params && .save`, I think we should be more specific.